### PR TITLE
Add OTel collector log file

### PR DESCRIPTION
**Pull request by dhurley**
### Proposed changes

Add OTel collector log file

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [ ] I have read the [`CONTRIBUTING`](https://github.com/nginx/agent/blob/main/docs/CONTRIBUTING.md) document
- [ ] I have run ```make install-tools``` and have attached any dependency changes to this pull request
- [ ] If applicable, I have added tests that prove my fix is effective or that my feature works
- [ ] If applicable, I have checked that any relevant tests pass after adding my changes
- [ ] If applicable, I have updated any relevant documentation ([`README.md`](https://github.com/nginx/agent/blob/main/README.md))
- [ ] If applicable, I have tested my cross-platform changes on Ubuntu 22, Redhat 8, SUSE 15 and FreeBSD 13

### CODE REVIEW

**Review by aphralG — APPROVED** [review]

**Comment by oliveromahony** [comment]
do we need a rotation policy to be added to the docs for this? like https://docs.nginx.com/nginx-agent/configuration/configuration-overview/#log-rotation

File: scripts/testing/load/Dockerfile [hunk 1]
```diff
@@ -81,6 +81,7 @@ ENV PATH=$PATH:/usr/local/go/bin
 
 RUN mv /agent/test/config/agent/nginx-agent-otel-load.conf /agent/test/load/nginx-agent.conf
 RUN mkdir /var/run/nginx-agent/
+RUN mkdir /var/log/nginx-agent/
```
Comment (oliveromahony) [inline scripts/testing/load/Dockerfile:84]: Can change to a single line?
```suggestion
RUN mkdir /var/log/nginx-agent/ /var/log/nginx-agent/
```

**Review by oliveromahony — APPROVED** [review]

**Comment by dhurley** [comment]
> do we need a rotation policy to be added to the docs for this? like https://docs.nginx.com/nginx-agent/configuration/configuration-overview/#log-rotation

That policy will cover the otel logs as well since its a regex that is used `/var/log/nginx-agent/*.log`

**Comment by oliveromahony** [comment]
> > do we need a rotation policy to be added to the docs for this? like https://docs.nginx.com/nginx-agent/configuration/configuration-overview/#log-rotation
> 
> That policy will cover the otel logs as well since its a regex that is used `/var/log/nginx-agent/*.log`

maybe an update to the docs to say that agent and otel logs are included as part of the policy? Just so the customers aren't surprised they got a bonus log file all of a sudden

**Pull request merged**